src/Server.js

```diff
@@ -1,16 +1,17 @@
 import dnode from 'dnode';
 import path from 'path';
-import WatchmanClient from './WatchmanClient';
+import sane from 'sane';
+import glob from 'glob';
 import { run as runLint } from './LintRunner';
 const CLIEngine = require('eslint').CLIEngine;
 
-const ROOT_DIR = '/Users/arthur/code/pinboard/webapp';
+const ROOT_DIR = process.cwd();
```

Comment:
ideally `esprint` works even if we run it from a subfolder. my initial idea is to have a .esprintconfig that marks the top level of a project. this could work for now.